src/main/java/org/springframework/integration/aws/inbound/SqsMessageDrivenChannelAdapter.java

```diff
@@ -92,6 +92,10 @@ public void setDestinationResolver(DestinationResolver<String> destinationResolv
 		this.simpleMessageListenerContainerFactory.setDestinationResolver(destinationResolver);
 	}
 
+	public void setAutoStartUp(boolean autoStartUp) {
+		this.simpleMessageListenerContainerFactory.setAutoStartup(autoStartUp);
```

Comment:
Great! 
Thank you for contribution!

Pay attention, please, that super class `AbstractEndpoint` have already `setAutoStartup()` method.
So, we should just override that here and call `super` anyway.
And yes, delegate the `autoStartup` flag to the `simpleMessageListenerContainerFactory`.
That is a bug.
Plus, we have to cover this change in the `SqsMessageDrivenChannelAdapterParserTests`, where the config is with `auto-startup="false"` already, but we don't check the option applied on the target `listenerContainer`.

And don't hesitate to add your name to `@author` list!